docs/attributes-dictionary/README.md

```diff
@@ -0,0 +1,27 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Registry
+weight: -2
+--->
+
+# Attributes Registry
```

Comment:
Is "Attributes Dictionary" (from the file name) the same as the "Attributes Registry" (from the section name)?

If so, we should settle for just one name.

Reply:
good point, will rename it